doc/source/development.rst

```diff
@@ -99,6 +99,50 @@ directory will take effect without reinstalling the package.
 
 .. warning:: if you run ``python setup.py install``, files will be copied from the Ray directory to a directory of Python packages (``/lib/python3.6/site-packages/ray``). This means that changes you make to files in the Ray directory will not have any effect.
 
+Building Ray on Windows (full)
+------------------------------
+
+**Requirements**
+
+The following links were correct during the writing of this section. In case the URLs changed, search at the organizations' sites.
+
+- bazel 3.2
+- Microsoft Visual Studio 2019 (or Microsoft Build Tools 2019 - https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019)
+- JDK 15 (https://www.oracle.com/java/technologies/javase-jdk15-downloads.html)
+- Miniconda 3 (https://docs.conda.io/en/latest/miniconda.html)
+- git for Windows, version 2.31.1 or later (https://git-scm.com/downloads/win)
+
+**Steps**
+
+1. Enable Developer mode on Windows 10 systems.
```

Comment:
Can you help explain why developer mode is needed? Some folks may be uncomfortable with it and explaining why might help alleviate that.

Reply:
Done.
I agree that Developer Mode shouldn't be necessary, but at the moment it is due to the use of symlinks in the source tree.
There is an open issue requesting symlinks to be removed (https://github.com/ray-project/ray/issues/15267).
If it is implemented, this step can be removed.